etc/config/circle.amazon.properties

```diff
@@ -1,17 +1,21 @@
-compilers=circlelatest:circle128
+compilers=circlelatest:circle129:circle128
 isSemVer=true
 defaultCompiler=circlelatest
 demangler=/opt/compiler-explorer/gcc-11.1.0/bin/c++filt
 objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
 
-#group.circle.intelAsm=-mllvm --x86-asm-syntax=intel
 compiler.circlelatest.exe=/opt/compiler-explorer/circle-latest/circle
 compiler.circlelatest.name=Latest
 compiler.circlelatest.semver=latest
 
+compiler.circle129.exe=/opt/compiler-explorer/circle-129/circle
```

Comment:
Thanks! In order for circle-129 to be installed, there's a PR needed to our https://github.com/compiler-explorer/infra repo (should be a super simple one to bin/yaml/circle.yaml only). I can do this if you're not comfortable doing so.

Reply:
Oh my! It's already done!! I'm so sorry I missed that! Merging now!